packages/neos-ui-ckeditor-bindings/src/EditorToolbar/style.css

```diff
@@ -19,6 +19,7 @@
     position: fixed;
     z-index: var(--zIndex-SecondaryToolbar-LinkIconButtonFlyout);
     width: 460px;
+    height: 56px;
```

Comment:
magic number. what is this number made of? could you `calc` it?

Reply:
Fixed, this is actually goldenUnit + 8px border-top + 8px border-bottom